src/components/shared/SearchInput.js

```diff
@@ -67,8 +67,8 @@ class SearchInput extends Component<Props> {
   }
 
   shouldShowErrorEmoji = () => {
-    const { count, query, showErrorEmoji } = this.props;
-    return showErrorEmoji && count === 0 && query.trim() !== "";
+    const { count, query } = this.props;
+    return count === 0 && query.trim() !== "" && !/^:\d*$/.test(query);
```

Comment:
So... I don't think our search input should know about go to line queries `!/^:\d*$/.test(query);`

Instead, it'd be better if we could use the `showErrorEmoji` flag so that QuickOpen can disable the error emoji when gotoline is set.

Reply:
I had suggested this in the other PR similar to this and I think I agree. Maybe we do one of two things if showErrorEmoji is provided it turns on whether we even show it. Next we either just use the count and query or we use a consumer provided boolean value.